### DISCUSSION TITLE
When I deploy to the server but in local work normally

### DISCUSSION THREAD

**Discussion by calakersky** [filamentphp/filament#11827] [Help]
### Package

Other

### Package Version

v3.2

### How can we help you?

![Screenshot 2024-03-13 043253](https://github.com/filamentphp/filament/assets/66521798/d35e207b-0d60-41aa-8495-4a38e1714277)

**Comment by leandrocfe** [comment]
Try this: https://discord.com/channels/883083792112300104/1217072970749579345/1217151892795162816
Are you using nginx?

**Reply by calakersky** [reply]
Sure, below is my Nginx config
error_page 404 /index.php?a=404;
location / {
  try_files $uri $uri/ /index.php?$query_string;
}

**Reply by leandrocfe** [reply]
Were you using v2, and did you upgrade to v3? Also, did you publish the Livewire config?
